src/web/components/StickyBottomBanner/StickyBottomBanner.tsx

```diff
@@ -24,28 +20,17 @@ const getBannerLastClosedAt = (key: string): string | undefined => {
 
 const DEFAULT_BANNER_TIMEOUT_MILLIS = 2000;
 
-const buildNewCmpBannerConfig = (): Banner => {
-    return {
-        id: 'cmpUi',
-        canShow: () => willShowCMP().then((result) => ({ result: !!result })),
+const buildCmpBannerConfig = (): Banner => ({
```

Comment:
@tjmw does this look ok to you? locally i'm not seeing any banners after accepting the privacy one, is this wrong or ok?

Reply:
talked offline